scipy/signal/lti_conversion.py

```diff
@@ -366,11 +366,12 @@ def cont2discrete(system, dt, method="zoh", alpha=None):
 
     Returns
     -------
-    sysd : tuple containing the discrete system
+    sysd : tuple containing the discrete system or bare instance of `lti`
         Based on the input type, the output will be of the form
 
-        * (num, den, dt)   for transfer function input
-        * (zeros, poles, gain, dt)   for zeros-poles-gain input
+        * bare instance of `lti` for `lti` input (in a tuple or not)
```

Comment:
It seems more consistent to return a tuple if given a tuple.  However, I'm not sure why length-1 tuples are intended to be supported anyway.

Reply:
I modified the code and docstring in order to return a dlti in a tuple if given a lti in a tuple (and a bare dlti if given a bare lti). I do not know why length-1 tuple should be supported either, but it appears as such in multiple places across the scipy-signal documentation. Thus, I do not have the power to decide interfering with this: this is a global scipy-signal decision and a core-designer of the package should take care of that kind of change.